meta-client/src/test/java/org/apache/helix/metaclient/impl/zk/TestMultiThreadStressTest/TestMultiThreadStressZKClient.java

```diff
@@ -217,13 +225,13 @@ public void testComplexParentListenerPuppy() {
     AtomicInteger globalChildChangeCounter = new AtomicInteger();
     ChildChangeListener childChangeListener = (changedPath, changeType) -> {
       globalChildChangeCounter.addAndGet(1);
-      System.out.println("-------------- Child change detected: " + changeType + " at path: " + changedPath + " number of changes: " + globalChildChangeCounter.get());
+      System.out.println("-------------- Child change detected: " + changeType
+          + " at path: " + changedPath + " number of changes: " + globalChildChangeCounter.get());
     };
-
-
     _zkMetaClient.subscribeChildChanges(zkParentKey, childChangeListener, false);
 
-    PuppySpec puppySpec = new PuppySpec(PuppyMode.REPEAT, 0.2f, new ExecDelay(5000, 0.1f), 5);
+    PuppySpec puppySpec = new PuppySpec(PuppyMode.REPEAT, 0.2f,
+        new ExecDelay(5000, 0.1f), 5);
```

Comment:
NIT: would suggest to have style, refactor based PR a separate one. It can distract the review from core logic.

Reply:
Sounds good, I'll remove style fixes from this PR, thanks!